cosmoz-moment.html

```diff
@@ -65,18 +84,24 @@
 		Polymer({
 			is: 'cosmoz-moment',
 			properties: {
+				/**
+				 * Locale abbreviation for Moment.js locale
+				 *
+				 * @attribute locale
+				 * @type string
+				 * @default 'en'
+				 */
 				locale: {
 					type: String,
+					value: function () {
```

Comment:
no need to run a function for String, if you don't, default value will be detectable, too